src/backend/history.js

```diff
@@ -3,15 +3,25 @@ export function trackHistory(hook, bridge) {
   const history = repo.history
 
   history.on('append', action => {
-    bridge.on(`toggle:${action.id}`, () => history.toggle(action))
-    bridge.on(`remove:${action.id}`, () => history.remove(action))
+    bridge.on(`toggle:${action.id}`, () => {
+      history.toggle(action)
+      // if toggling an inactive action, history won't reconcile
+      // we still want to know what history looks like though
+      bridge.send('history:reconcile', JSON.stringify(history))
+    })
+
+    bridge.on(`remove:${action.id}`, () => {
+      history.remove(action)
+      bridge.send('history:reconcile', JSON.stringify(history))
```

Comment:
This is necessary because, when you delete a node not within the active branch, a reconciliation doesn't occur?